dirs/dirs.go

```diff
@@ -349,8 +349,8 @@ func SetRootDir(rootdir string) {
 		SystemFontconfigCacheDirs = append(SystemFontconfigCacheDirs, filepath.Join(rootdir, "/usr/lib/fontconfig/cache"))
 	}
 
-	FreezerCgroupDir = filepath.Join(rootdir, "/sys/fs/cgroup/freezer/")
-	PidsCgroupDir = filepath.Join(rootdir, "/sys/fs/cgroup/pids/")
+	CgroupDir = filepath.Join(rootdir, "/sys/fs/cgroup/")
```

Comment:
should this one be exported? I  don't see it used but maybe I missed that

Reply:
It used to be but after things moved to `sandbox/cgroup` it ended up being orphaned. I think it needs to be revisited, maybe it is obsolete now.

Reply:
As suggested, it is not used anymore. This is no longer exported.